### PR TITLE
Add fingerprint for Kia Forte GT 2021.

### PR DESCRIPTION
**Pull request by sobomax**
Add fingerprint for Kia Forte GT 2021.

**Checklist**
- [*] added to README
- [ ] test route added to [test_car_models](../../selfdrive/test/test_car_models.py)
- [*] route with openpilot: https://my.comma.ai/useradmin/?onebox=50a2212c41f65c7b%7C2021-01-18--18-36-45

### CODE REVIEW

**Comment by sobomax** [comment]
hmm self test is failing, ought to investigate...

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -141,6 +141,9 @@ class Buttons:
   }],
   CAR.KIA_FORTE: [{
     67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 356: 4, 544: 8, 593: 8, 608: 8, 688: 5, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 903: 8, 909: 8, 916: 8, 1040: 8, 1042: 8, 1078: 4, 1107: 5, 1136: 8, 1156: 8, 1170: 8, 1173: 8, 1191: 2, 1225: 8, 1265: 4, 1280: 4, 1287: 4, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1384: 8, 1394: 8, 1407: 8, 1427: 6, 1456: 4, 1470: 8
+  },
+  {
+    67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 354: 3, 356: 4, 544: 8, 576: 8, 593: 8, 608: 8, 688: 5, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 903: 8, 905: 8, 909: 8, 916: 8, 1040: 8, 1042: 8, 1056: 8, 1057: 8, 1078: 4, 1107: 5, 1136: 8, 1156: 8, 1170: 8, 1173: 8, 1186: 2, 1191: 2, 1225: 8, 1265: 4, 1280: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1384: 8, 1394: 8, 1407: 8, 1414: 3, 1419: 8, 1427: 6, 1456: 4, 1470: 8, 1988: 8, 1996: 8, 2000: 8, 2004: 8, 2008: 8, 2012: 8, 2015: 8
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:146]: this is a superset of the existing fingerprint, so just replace the existing one with the new one

Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:146]:
```suggestion
    67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 354: 3, 356: 4, 544: 8, 576: 8, 593: 8, 608: 8, 688: 5, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 903: 8, 905: 8, 909: 8, 916: 8, 1040: 8, 1042: 8, 1056: 8, 1057: 8, 1078: 4, 1107: 5, 1136: 8, 1156: 8, 1170: 8, 1173: 8, 1186: 2, 1191: 2, 1225: 8, 1265: 4, 1280: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1384: 8, 1394: 8, 1407: 8, 1414: 3, 1419: 8, 1427: 6, 1456: 4, 1470: 8, 1988: 8, 1996: 8, 2000: 8, 2004: 8, 2008: 8, 2012: 8, 2015: 8
```

File: README.md [hunk 2]
```diff
@@ -156,7 +156,7 @@ Community Maintained Cars and Features
 | Hyundai   | Veloster 2019                 | SCC + LKAS        | Stock            | 5mph               | 0mph         |
 | Jeep      | Grand Cherokee 2016-18        | Adaptive Cruise   | Stock            | 0mph               | 9mph         |
 | Jeep      | Grand Cherokee 2019-20        | Adaptive Cruise   | Stock            | 0mph               | 39mph        |
-| Kia       | Forte 2018-19                 | SCC + LKAS        | Stock            | 0mph               | 0mph         |
+| Kia       | Forte 2018-19,21              | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```
Comment (adeebshihadeh) [inline README.md:159]:
```suggestion
| Kia       | Forte 2018-19, 2021           | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/hyundai/values.py:146]: this will fix the failing test, new fingerprint is a superset of the existing fingerprint

**Comment by sobomax** [comment]
Thanks for a review @adeebshihadeh! Let me flatten the histrory and I'll force-push the branch.

**Comment by sobomax** [comment]
@adeebshihadeh cleaned and ready to land I think. Thanks!

**Comment by adeebshihadeh** [comment]
Thanks for the PR. In the future, no need to flatten the history, we always squash before merging.

**Pull request merged**